### PR TITLE
[RM CH07 ] Interchange 7.4 with 7.3

### PR DESCRIPTION
**Pull request by ASawwaf**
Interchange 7.4 ( scope ) with 7.3

### CODE REVIEW

File: doc/ref_model/chapters/chapter07.md [hunk 1]
```diff
@@ -87,8 +87,68 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor i
...
+<a name="7.3.2"></a>
+## 7.3.2 Security Requirements
+
+The following diagram shows the different security domains that impact the Reference Model:
+
+<p align="center"><img src="../figures/ch7_security_posture.png" alt="Overview" title="Security Domains" width="100%"/></p>
+<p align="center"><b>Figure 7-3:</b> Reference Model Security Domains</p>
+
+<a name="7.3.3"></a>
+## 7.3.3 Platform security requirements
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter07.md:117]: Issue #1676 suggest that we move this under Section 7.5

File: doc/ref_model/chapters/chapter07.md [hunk 2]
```diff
@@ -87,8 +87,68 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor i
...
+* Strong separation between tenants - ensuring network, data, and runtime process isolation between tenants
+* Authenticated/secure APIs provided to overlay network administrators
+* Platform change control on hardware
+* Templated approved changes for automation where available
+* Typically well defined security framework documentation including approved deployment use cases
+* Infrastructure software update process
+* Identity Domain = platform
+
+<a name="7.3.4"></a>
+## 7.3.4 Workload security requirements
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter07.md:133]: Issue #1676 suggest that we move this under Section 7.6

File: doc/ref_model/chapters/chapter07.md [hunk 3]
```diff
@@ -87,8 +87,68 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor i
...
+* Up to platform-level certification
+* Each workload network will need to undertake it own security self-assessment and accreditation, and not inherit a security accreditation from the platform
+* Potentially automated service activation
+* Workload owner owns workload security certification process
+* Workload owner owns workload design change process
+* Workload owner owns workload software update process
+* Identity Domain = workload
+
+<a name="7.3.5"></a>
+## 7.3.5 Certification/validation requirements
```
Comment (pgoyal01) [inline doc/ref_model/chapters/chapter07.md:145]: Issue #1676 suggested that we delete this

**Comment by ASawwaf** [comment]
@pgoyal01 , my plan to have it in 2 PR , Merge this and after that move it with other PR

**Comment by ASawwaf** [comment]
@walterkozlowski , I can not discover it :(

**Comment by ASawwaf** [comment]
@walterkozlowski @rabi-abdel @CsatariGergely @deebhatia @cntt-n/ref-m 

Plz approve

**Comment by rabi-abdel** [comment]
@walterkozlowski as WSL, this require your approval to get merged.

**Pull request merged**